src/console/c_cvars.h

```diff
@@ -416,14 +416,13 @@ FBaseCVar *cvar_forceset (const char *var_name, const char *value);
 inline FBaseCVar *cvar_set (const char *var_name, const uint8_t *value) { return cvar_set (var_name, (const char *)value); }
 inline FBaseCVar *cvar_forceset (const char *var_name, const uint8_t *value) { return cvar_forceset (var_name, (const char *)value); }
 
-
-
 // Restore demo cvars. Called after demo playback to restore all cvars
 // that might possibly have been changed during the course of demo playback.
 void C_RestoreCVars (void);
 
 void C_ForgetCVars (void);
 
+void C_GrabCVarDefaults (); // parse DEFCVARS
```

Comment:
This function is used only once, so it would be better to use a forward declaration at the location of its call. IMHO changes to header are redundant here.

Reply:
I really wasn't too sure about this one. Nevertheless, for selfish reasons I'll just assume you're right. To be quite honest, this is a header file that if changed, causes nearly the entire source to recompile, so not changing this header actually makes me really really happy.